mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,159 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	you can use this feature in a negated boolean-context fashion:
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+High dynamic range (HDR) represents the combination of brightness, color depth, transfer function, and color gamut.
+
+<dl dfn-type=value dfn-for="dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered high
+		* color depth is greater than 24 bit
+		* has a <a href="#contrast-of-display">high contrast ratio</a>
+	<dt><dfn>standard</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered standards
+		* color depth is 24 bit or 8 bit per color component of RGB
+		* has a <a href="#contrast-of-display">low contrast ratio</a>
+</dl>
+
+<h3 id="contrast-brightness-of-display">
+	Determining contrast and brightness of display</h3>
+
+This text is non-normative. There currently is no agreed upon standardization of
+high contrast ratio, low contrast ratio, max brightness or low brightness as it
+relates to displays. As such, the determination for <a>dynamic-range</a> and
+<a>video-dymanic-range</a> will be defined by the user agent.
+
+<h3 id="video-prefixed-properties">
+Handling bi-plane scenarios</h3>
```

Comment:
I like "Video Prefixed Properties". Purely subjective. 

I was curious to see what this would look like relative to the other features, so I dumped the raw text into https://api.csswg.org/bikeshed/, but this produces some build errors that stopped the output from generating. LMK if I'm holding it wrong.

Reply:
The bikeshed compile issues have been fixed. You can try again.